x-pack/plugins/cases/common/types/api/configure/v1.ts

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Just as a reminder we'll want to switch the file names and anywhere we have `v1` to some date.

Reply:
Thanks, I will!

Reply:
We talked about it offline and we decided to keep it as it is.